### PR TITLE
📖 amp-consent: Add Ogury CMP doc + example

### PR DESCRIPTION
**Pull request by PierigOgury**
/cc @zhouyx

### CODE REVIEW

**Comment by zhouyx** [comment]
Thank you @PierigOgury for the integration!

I played with the example you provided and noticed that the prompt dialog was expanded to fullscreen right away. Is this a general pattern you adopt elsewhere? That the prompt always takes the entire screen. Thanks

**Comment by yopfull** [comment]
@zhouyx yes, it is the pattern we currently use everywhere

**Comment by zhouyx** [comment]
Add @brianlzw

@yopfull Thank you for the information. The AMP team did some investigation on the UX again. And we believe that prompt to fullScreen immediately is not a good user experience. It also introduces some security concern as the prompt can take up the entire space of the publishers site. 
To address that we've decided to only allow full screen with user interactions. It won't work with the example you provided. Please let us know what do you think. Thank you!

**Comment by yoannOgury** [comment]
Hi @zhouyx,

Thank you for the feedback. I answer to your previous comment as @PierigOgury is on vacation.

We had initially planned to change the UI so that the consent notice can take less space in a bit more time but we will advance a bit this task.
We have in mind to take 60% of the view height and display a blocking grey overlay with a transparency to still show the page behind. Quite close to the example "Advanced User Consent Flow".
With this update the user is still required to give an answer to the notice but is still viewing the page behind.
Do you have any thoughts on this proposition?

Thanks!

**Comment by zhouyx** [comment]
The reason we recommend 30vh is because the prompt won't be blocking, so that we think it leads to better user experience. But I do understand that it's also a common pattern to have the prompt block the content for better engagement. 
Personally I find the 60vh solution better than full screen, because it resolves the security concern. @brainlzw what's your thoughts on the initial 60vh blocking prompt (with overlay) in the UX perspective? Thank you

**Comment by zhouyx** [comment]
FYI: Here's the tracking issue #26197

**Comment by yopfull** [comment]
@zhouyx  the behavior of our cmp has been updated has expected with some design update.
Now by default the height of the consent notice is  30vh, with an overlay. The fullscreen mode is activated only after user interaction (fullscreen button click).

**Comment by PierigOgury** [comment]
@zhouyx the behavior of our cmp has been updated has expected with some design update.
Now by default the height of the consent notice is 30vh, with an overlay. The fullscreen mode is activated only after user interaction (fullscreen button click).

File: examples/amp-consent/cmp-vendors.amp.html [hunk 1]
```diff
@@ -205,6 +206,28 @@ <h3>Image that is NOT blocked by consent</h3>
...
+      <!-- Ogury example -->
+      <amp-consent id='ogury' layout='nodisplay' type='Ogury'>
+        <script type="application/json">
+          {
+            "postPromptUI": "custom-prompt-ui",
+            "clientConfig": {
+              "assetKey": "OGY-316B32F7E5B7"
+            },
+            "uiConfig": {
+              overlay: true
```
Comment (zhouyx) [inline examples/amp-consent/cmp-vendors.amp.html]: nit: invalid JSON. "overlay"

**Comment by zhouyx** [comment]
Thank you for the UI change. Looks great.

File: examples/amp-consent/cmp-vendors.amp.html (hunk 1 above)
Comment (PierigOgury) [reply examples/amp-consent/cmp-vendors.amp.html]: @zhouyx Indeed, it's fixed

**Comment by zhouyx** [comment]
@PierigOgury travis is red because a deadlink is detected. I've merged #26707 to fix it. Could you please sync to latest master and then I can proceed and merge your PR. Thank you

**Comment by yopfull** [comment]
@zhouyx thanks, all checks are green the PR looks ready t o merge

**Comment by zhouyx** [comment]
PR Merged! 🚢 🎉

**Pull request merged**